app/services/langfuse/tasks/alert_summary/summary.py

```
@@ -0,0 +1,185 @@
+
+            # Mark summary as sent for today
+            if success and is_redis_configured():
+                try:
+                    redis_service = await get_redis_service()
+                    await redis_service.setex(
+                        summary_sent_key, 86400, "1"
+                    )  # 24 hour TTL
+                    logger.info(
+                        f"Daily alert summary sent successfully for {yesterday}"
+                    )
+                except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)